<launch>
<node name="LD06" pkg="ldlidar" type="ldlidar" output="screen" >
<param name="topic_name" value="LiDAR/LD06"/>
<param name="port_name" value ="/dev/ttyS0"/>
<param name="frame_id" value="lidar_frame"/>
</node>
</launch>
